Sources/Client/Model/Channel/Channel+Requests.swift

```diff
@@ -133,6 +135,37 @@ public extension Channel {
         Client.shared.send(eventType: eventType, to: self, completion)
     }
     
+    /// Send a keystroke event for the current user.
+    /// - Parameter completion: a completion block with `Event`.
+    @discardableResult
+    func keystroke(_ completion: @escaping Client.Completion<Event>) -> Cancellable {
+        currentUserTypingTimerControl?.cancel()
+        
+        currentUserTypingTimerControl = DefaultTimer.schedule(timeInterval: 5, queue: .main) { [weak self] in
+            self?.stopTyping(completion)
+        }
```

Comment:
It would be nice to extract the `5` into a preset which could be changed by a user.

Reply:
Let's leave it for now with a value from the task description.